src/H5Zzfp.c

```diff
@@ -47,11 +47,7 @@ and calls to bitstream methods with 'B ' as in
 #include "H5Zzfp_plugin.h"
 #include "H5Zzfp_props_private.h"
 
-/* Convenient CPP logic to capture ZFP lib version numbers as compile time string and hex number */
-#define ZFP_VERSION_STR__(Maj,Min,Rel) #Maj "." #Min "." #Rel
-#define ZFP_VERSION_STR_(Maj,Min,Rel)  ZFP_VERSION_STR__(Maj,Min,Rel)
-#define ZFP_VERSION_STR                ZFP_VERSION_STR_(ZFP_VERSION_MAJOR,ZFP_VERSION_MINOR,ZFP_VERSION_RELEASE)
-
+/* Convenient CPP logic to capture ZFP lib version numbers as compile time hex number */
 #define ZFP_VERSION_NO__(Maj,Min,Rel)  (0x0 ## Maj ## Min ## Rel)
```

Comment:
This likely won't work well with zfp's new versioning system as of 1.0.0.  With this latest release, we suggest using `ZFP_MAKE_VERSION`.

Reply:
I will deal with when I upgrade to 1.0.0. leaving as is for this PR